src/Build/Construction/ProjectImportElement.cs

```diff
@@ -60,144 +95,341 @@ public string Project
         }
 
         /// <summary>
-        /// Location of the project attribute
+        ///     Location of the project attribute
         /// </summary>
         public ElementLocation ProjectLocation => GetAttributeLocation(XMakeAttributes.project);
 
+#nullable enable
+
         /// <summary>
-        /// Gets or sets the SDK that contains the import.
+        ///     Gets or sets the SDK that contains the import.
         /// </summary>
-        public string Sdk
+        public string? Sdk
         {
-            get => FileUtilities.FixFilePath(GetAttributeValue(XMakeAttributes.sdk));
```

Comment:
Behavioral change there. I feel like this was just copy-pasted from `Project` property. It makes no sense to call `FixFilePath` here, since `Sdk` is not a file path (it can't even contain any slashes).